src/pretix/presale/views/waiting.py

```diff
@@ -89,6 +123,11 @@ def get(self, request, *args, **kwargs):
             r._csp_ignore = True
             return r
 
+        if not self.itemvars:
+            messages.success(request, _("No ticket types are available for the waiting list, have a look at the "
```

Comment:
ich würde eher eine `info` nehmen, weil so richtig was erfolgreiches passiert ist ja noch nicht in dem moment?
```suggestion
            messages.info(request, _("No ticket types are available for the waiting list, have a look at the "
```

Reply:
Ich versuche messages.info zu meiden, weil wir alert-info eigentlich sowieso möglichst spärlich und wenn, eher für "anhaltende zustände" statt als reaktionen auf einen klick einsetzen… aber ja, hier passt es wohl besser…